MantisPhpClient.php

```diff
@@ -671,11 +712,33 @@ private function getUserPreference( $p_preference ) {
  * Filter class used to capture information about a single filter.
  */
 class Filter {
+  /**
+   * @var int
+   */
     public $id;
+
+  /**
+   * @var string
+   */
     public $name;
 
+  /**
+   * Filter constructor.
+   * @param int $id
+   * @param string $name
+   */
     public function __construct( $id, $name ) {
         $this->id = $id;
         $this->name = $name;
     }
 }
+
+if(!function_exists('lang_get_text')){
+  /**
+   * @param string $str
+   * @return string
+   */
+  function lang_get_text($str){
```

Comment:
Instead of defining this.  Let's just use the English values directly in getStandardFiltersList.  If the client wants localization, then can do the localization later based on filter.id or just use the English name via filter.name.